rtt_rosparam/README.md

```diff
@@ -53,6 +53,7 @@ policy (see below).
 ##### Operations for getting single properties
 
 * **getParam(ros_name, rtt_name)** Get the ROS param **ros_name** and store it in the RTT property **rtt_name**. Use leaders like `~` and `/` for private and absolute resolution.
+* `getParam_type_(name)` Gets the value of a ROS param named `name`. The value returned is the default constructor if no `name` parameter can be found. Use leaders like `~` and `/` for private and absolute resolution.
```

Comment:
```suggestion
* `getParam<Type>(name)` Gets the value of a ROS param named `name` of type `<type>`. The value returned is the default constructor if no `name` parameter can be found. Use leaders like `~` and `/` for private and absolute resolution.
```

But it does not fit in this category because it does not get a parameter value and stores it in a property:
> ##### Operations for getting single properties